code/modules/admin/sql_message_system.dm

```diff
@@ -629,6 +629,19 @@
 	browser.set_content(jointext(output, ""))
 	browser.open()
 
+/// Represents a message stored in the db
+/datum/admin_message
```

Comment:
should this be admin-VV-protected or does it not matter

Reply:
it's basically just a struct, exists to hold data with names. it shouldn't really matter, even if an admin does make their own the most they could do is display a message in chat which they can already do with debug tools

Reply:
bokay